app/views/application/index.html

```diff
@@ -114,6 +114,22 @@ <h2 class="govuk-heading-m govuk-!-font-size-27">Evidence of expertise</h2>
           {% endif %}
         </ul>
       </section>
+
+      <section class="govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-m govuk-!-font-size-27">Competencies/Soft skills</h2>
+        <ul class="app-task-list">
+          {{ appTaskListItem({
+            name: 'Communication, analytical and judgement skills',
+            href: 'application/competencies/add-communication-details',
+            taskName: data.conflictOfInterestCompleted
```

Comment:
this name needs to match the data name from the 'section-completed.html file' `data.communicationCompleted`